tests/IVIMmodels/unit_tests/test_ivim_fit.py

```diff
@@ -114,24 +114,31 @@
 
 def signal_helper(signal):
     signal = np.asarray(signal)
-    signal = np.abs(signal)
-    signal /= signal[0]
-    ratio = 1 / signal[0]
-    return signal, ratio
+    signal = np.abs(signal) #Oliver: do we need this for unit testing? It will bias the fits by forcing positive signals.
+    signal /= signal[0] # this scales noise differenly. I have now made sure signal is produced with amplitude 1 if right flags are selected. Then  SNR is fixed. Then, this sentence may be redundant
+    #ratio = 1 / signal[0] #this is 1 per definition (See sentence above)
+    return signal
 
-def tolerances_helper(tolerances, ratio, noise):
+def tolerances_helper(tolerances, data):
+    epsilon = 0.0001
     if "dynamic_rtol" in tolerances:
         dyn_rtol = tolerances["dynamic_rtol"]
-        scale = dyn_rtol["offset"] + dyn_rtol["ratio"]*ratio + dyn_rtol["noise"]*noise + dyn_rtol["noiseCrossRatio"]*ratio*noise
-        tolerances["rtol"] = {"f": scale*dyn_rtol["f"], "D": scale*dyn_rtol["D"], "Dp": scale*dyn_rtol["Dp"]}
+        scale = dyn_rtol["offset"] + dyn_rtol["noise"]*data["noise"]
+        if dyn_rtol["scale_f"]:
```

Comment:
I think this is a misnomer and should be renamed. I thought something called `scale_f` would be scaling the `f` parameter. Instead it scales the others by `f`.